src/components/ProgramList.tsx

```diff
@@ -0,0 +1,165 @@
+            format: "online",
+            tag: "math",
+        },
+        {
+            name: "placeholder",
+            image: "",
+            description: "",
+            startDate: "",
+            endDate: "",
+            format: "",
+            tag: "",
+        },
+        {
+            name: "placeholder",
+            image: "",
+            description: "",
+            startDate: "",
+            endDate: "",
+            format: "",
+            tag: "",
+        },
+    ];
+
+    return (
+        <Center>
+            <Wrap spacing="50px" justify="space-between">
+                {imagesAndDescriptions.map((item, idx) => {
+                    return item.name == "placeholder" ? (
+                        <WrapItem
+                            flexBasis="300px"
+                            flexGrow={1}
+                            display="hidden"
+                            key={idx}
+                            _hover={{
+                                borderColor: "gray.600",
+                                borderWidth: 1,
+                            }}
+                        ></WrapItem>
+                    ) : (
+                        <WrapItem
+                            flexBasis="300px"
+                            key={idx}
+                            flexGrow={1}
+                            _hover={{
+                                borderColor: "gray.600",
+                                borderWidth: 1,
+                            }}
+                        >
+                            <Box borderWidth="1px" width="100%">
+                                <AspectRatio width="100%" ratio={4 / 2}>
+                                    <Image
+                                        src={item.image}
+                                        boxSize="200"
+                                        key={idx}
+                                        data-index={idx}
+                                        fit="cover"
+                                        alt={item.name}
+                                    />
+                                </AspectRatio>
+
+                                <Box p="6">
+                                    <Box d="flex" alignItems="baseline">
+                                        <Box
+                                            mt="1"
+                                            fontWeight="semibold"
+                                            as="h3"
+                                            lineHeight="tight"
+                                            isTruncated
+                                        >
+                                            {item.name}
+                                        </Box>
+                                        <Spacer />
+                                        <Badge
+                                            borderRadius="full"
+                                            padding="1"
```

Comment:
I played around a bit with the padding and I think the following paddings look kinda similar-ish to the figma design:
- `pb="1"`
- `pt="1.5"`
- `px="3"`
<img width="1062" alt="Screen Shot 2021-07-22 at 11 56 59 PM" src="https://user-images.githubusercontent.com/42329517/126735753-e4937791-f359-434b-8c5d-d21bafe6e2d5.png">

Reply:
![image](https://user-images.githubusercontent.com/50597354/126857937-070594de-ef74-4614-9900-0f566af41a9d.png)
Wow thanks! They are indeed thicker :D